index.html

```diff
@@ -26,6 +26,159 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section>
```

Comment:
Your works section should be wrapped in a `section` tag to follow a proper structure and hierarchy due to semantic reasons 👀 

    <section>
        // project card goes here
        // project card goes here
        ...
    </section>

Kindly make these changes so that we meet the project requirements 📝

Also, please be sure that you use `display: grid` on the main section in order to position your project cards as it is a project requirement 😉